raizama/core/management/commands/importspreadsheet.py

```diff
@@ -0,0 +1,110 @@
+import csv
+import re
+from django.core.management.base import BaseCommand
+from raizama.core.models import Organization
+from pathlib import PurePath
+
+
+class Command(BaseCommand):
+    help = "Import .csv file with organization's data and update the database."
+
+    def add_arguments(self, parser):
+        parser.add_argument("file-path", type=str, help="Path of a .csv file")
+
+    def check_duplicate(self, name, city):
+        queryset_orgs = Organization.objects.values("name", "city")
+        return not queryset_orgs.filter(
+            name__iexact=name, city__iexact=city
+        ).exists()
+
+    def extract_social_nets(self, social_organization):
+        social_nets = social_organization.replace("\n", " ")
+        facebook_identifier = "F:"
+        instagram_identifier = "I:"
+        match_f = re.search(facebook_identifier, social_nets)
+        match_i = re.search(instagram_identifier, social_nets)
+
+        if match_f and match_i:
+            if match_f.span()[0] < match_i.span()[0]:  # facebook comes first
+                social_nets_tuple = social_nets.rpartition(instagram_identifier)
+                facebook_organization = social_nets_tuple[0].replace(
+                    facebook_identifier, ""
+                )
+                facebook_organization = facebook_organization.replace(" ", "")
+                instagram_organization = social_nets_tuple[2]
+                instagram_organization = instagram_organization.replace(" ", "")
+                return facebook_organization, instagram_organization
+
+            social_nets_tuple = social_nets.rpartition(facebook_identifier)
+            instagram_organization = social_nets_tuple[0].replace(
+                instagram_identifier, ""
+            )
+            instagram_organization = instagram_organization.replace(" ", "")
+            facebook_organization = social_nets_tuple[2]
+            facebook_organization = facebook_organization.replace(" ", "")
+            return facebook_organization, instagram_organization
+
+        if match_f and not match_i:
+            facebook_organization = social_nets.replace(facebook_identifier, "")
+            facebook_organization = facebook_organization.replace(" ", "")
+            instagram_organization = ""
+            return facebook_organization, instagram_organization
+
+        if not match_f and match_i:
+            instagram_organization = social_nets.replace(instagram_identifier, "")
+            instagram_organization = instagram_organization.replace(" ", "")
+            facebook_organization = ""
+            return facebook_organization, instagram_organization
+
+        if not match_f and not match_i:
+            instagram_organization = ""
+            facebook_organization = ""
+            return facebook_organization, instagram_organization
+
+    def extract_fields(self, row_organization):
```

Comment:
O que esse método faz é o que chamamos de _serializar_. Que tal chamar ele de `serialize`? Além disso, em inglês seria _organization row_, então eu renomearia tb o nome do argumento (ou simplificaria apenas para `row`:

```suggestion
    def serialize(self, row):
```

O mesmo eu faria com outras variáveis (por exemplo, pq `address_organization` e não só `address`?). Assim deixamos mais curtos e objetivos os nomes, e evitamos inconsistências como `address_organization` (por extenso) e `instagram_org` (abreviada).